contracts/MarketMaker.sol

```diff
@@ -15,8 +16,12 @@ contract MarketMaker is Ownable, IERC1155TokenReceiver {
     /*
      *  Events
      */
-    event AutomatedMarketMakerFunding(uint funding);
-    event AutomatedMarketMakerClosing();
+    event AutomatedMarketMakerCreated(uint funding);
+    event AutomatedMarketMakerPaused();
+    event AutomatedMarketMakerResumed();
+    event AutomatedMarketMakerClosed();
+    event MarketFundingChanged(int funding);
```

Comment:
For `MarketFundingChanged`, rename `funding` (see later comment).

Also, for consistency, `AutomatedMarketMakerFundingChanged`. *Or*, to leave off the typing, say `AMMCreated`, `AMMPaused`, etc. This repo *is* called `hg-automated-market-makers` after all.